addons/web/static/src/js/views/list/list_renderer.js

```diff
@@ -228,6 +231,19 @@ var ListRenderer = BasicRenderer.extend({
         var n = this.columns.length;
         return this.hasSelectors ? n + 1 : n;
     },
+    /**
+     * Returns the local storage key for stored enabled optional columns
+     *
+     */
+    _getOptionalColumnsStorageKey: function () {
+        var self = this;
+        var fields = [];
+        _.each(this.state.fieldsInfo[this.viewType], function (field, name) {
+            fields.push(name + ":" + self.state.fields[name].type);
+        });
+        fields.sort();
```

Comment:
Not sure this is necessary. We have to talk about this function anyway

Reply:
@aab-odoo Not remember exactly but when I was working with this branch I intentionally added this